accounts/tests.py

```diff
@@ -1,3 +1,29 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User
+from .models import UserAccount
 
-# Create your tests here.
+
+@pytest.fixture
+def test_account(db):
```

Comment:
Fixtures names should not start with "test_", only actual tests start like that.
You can call this fixture "user_account" or something like that.